share/normalize/links.py

```diff
@@ -764,10 +768,14 @@ class URLLink(AbstractIRILink):
     def hint(cls, obj):
         if cls.URL_RE.search(obj) is not None:
             return 0.25
+        if obj[:4].lower() == 'www.':
```

Comment:
While you're at it would you mind adding `ww2.` as well? I've seen that floating around some places.